### PR TITLE
9 ➡️ 12

### PR DESCRIPTION
**Pull request by jennuine**
# ➡️ Forward port

Port `sdf9` to `sdf12`

Branch comparison: https://github.com/gazebosim/sdformat/compare/sdf12...sdf9

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

File: cmake/SearchForStuff.cmake [hunk 1]
```diff
@@ -0,0 +1,152 @@
+include (FindPkgConfig)
```
Comment (scpeters) [inline cmake/SearchForStuff.cmake:1]: we removed this `SearchForStuff.cmake` file when migrating to `ign-cmake2`. We don't want to re-add it. Let's try to figure out which pull request induced a change here and figure out how to apply that change elsewhere (probably the root CMakeLists.txt)

**Review by scpeters — CHANGES_REQUESTED** [review]
don't add SearchForStuff.cmake

File: cmake/SearchForStuff.cmake (hunk 1 above)
Comment (scpeters) [reply cmake/SearchForStuff.cmake:1]: looks like the changes were made in https://github.com/gazebosim/sdformat/pull/1042

Comment (jennuine) [reply cmake/SearchForStuff.cmake]: 9c81632bca096b86655a0501ca9e47be278ac609

**Review by scpeters — APPROVED** [review]

**Comment by scpeters** [comment]
thanks!

**Pull request merged**